website/docs/docs/get-started/dbt-cloud-features.md

```diff
@@ -3,44 +3,109 @@ title: "dbt Cloud features"
 id: "dbt-cloud-features"
 ---
 
+dbt Cloud is the fastest and most reliable way to deploy dbt. Develop, test, schedule, document, and investigate data models all in one web-based UI. 
 
-## dbt Cloud IDE
+In addition to providing a hosted architecture for running dbt Core across your organization, dbt Cloud comes equipped with turnkey support for scheduling jobs, CI/CD, hosting documentation, monitoring & alerting, and an integrated developer environment (IDE).
 
+dbt Cloud's [flexible plans](https://www.getdbt.com/pricing/) and features make it well-suited for data teams of any size &mdash; sign up for your [free 14 day trial](https://www.getdbt.com/signup/)! <br></br>
 
-The dbt Cloud [integrated development environment (IDE)](/docs/get-started/develop-in-the-cloud) allows you to build, test, run, and version control your dbt projects directly from your browser. The IDE is the fastest and most reliable way to deploy dbt, and provides a real-time editing and execution environment for your dbt project &mdash; no command line required.  Anyone can use the IDE, from new dbt developers to seasoned practitioners. 
 
-With the Cloud IDE, you can:
+<div className="grid--2-col">
```

Comment:
@mirnawong1 I love the card on this page! One problem is that you have to scroll a lot to them all. How does a 3-column view look? You might consider that.

Reply:
Oh I did tried this already (3 column) but I scrapped it because it made the card width too narrow. I'll share a screenshot in a sec. 

also thi is one of those situations where hiding the leftside bar would be great (if the page auto formats ofc)